akka-remote/src/main/resources/reference.conf

```diff
@@ -98,6 +98,11 @@ akka {
 
     # Acknowledgment timeout of management commands sent to the transport stack.
     command-ack-timeout = 30 s
+    
+    # The timeout for outbound associations to perform the handshake.
+    # If the transport is akka.remote.netty.tcp or akka.remote.netty.ssl
+    # the configured connection-timeout for the transport will be used instead.
```

Comment:
I was thinking that otherwise the user would always have to change both those settings. When would you use different timeout here than the connection-timeout?